example/workflow/webview/src/app.ts

```diff
@@ -16,8 +16,9 @@
 import { createWorkflowDiagramContainer } from '@eclipse-glsp-examples/workflow-glsp';
 import { GLSPDiagramIdentifier, GLSPStarter } from '@eclipse-glsp/vscode-integration-webview';
 import '@eclipse-glsp/vscode-integration-webview/css/glsp-vscode.css';
-import { Container } from 'inversify';
+import '@vscode/codicons/dist/codicon.css';
 
+import { Container } from 'inversify';
```

Comment:
The empty line before and missing empty line after the inversify import is a bit weird.